gaphor/ui/filemanager.py

```diff
@@ -334,9 +333,12 @@ def async_saver():
                     window=self.parent_window,
                 )
                 raise
+            else:
+                time.sleep(2)
+                self.filename = filename
+                self._update_monitor()
```

Comment:
This should be handled via `GLib.timeout_add`. This way the whole UI freezes.

Reply:
in the end, the timeout really did not fully resolve this... I still saw the popups. Therefore, I just ran Gaphor with debugger and realized that the popups on Mac are triggered specifically by ATTRIBUTE_CHANGED events, which I don't think are events we're interested in. As such, I've now added a check against this event-type, which resolves this issue. Happy to hear your thoughts on this!